### PR TITLE
Facebook pixel analytic

### PR DESCRIPTION
**Pull request by VojtaSim**
Implementation of Facebook Pixel analytic on IMA.js Analytic interface

Depends on: https://github.com/seznam/IMA.js-plugins/pull/6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,9 @@
 
 The IMA.js plugins is multi-package repository containing most commonly used plugins on IMA.js application development stack.
 
-- [**plugin-analytic-google**](packages/plugin-analytic-google)
 - [**plugin-analytic**](packages/plugin-analytic)
+- [**plugin-analytic**](packages/plugin-analytic-fb-pixel)
```
Comment (mjancarik) [inline README.md]: [**plugin-analytic-fb-pixel**]

File: packages/plugin-analytic-fb-pixel/README.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+		analytic: {
+			fbPixel: {
+				id: 'XXX'
+			}
+		}
+	}
+}
+```
+
+```javascript
```
Comment (mjancarik) [inline packages/plugin-analytic-fb-pixel/README.md]: Why must you create alias?

File: packages/plugin-analytic-fb-pixel/src/main.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import { ScriptLoaderPlugin } from 'ima-plugin-script-loader';
+import FacebookPixelAnalytic from './FacebookPixelAnalytic';
+
+let defaultDependencies = [
```
Comment (mjancarik) [inline packages/plugin-analytic-fb-pixel/src/main.js]: The property $dependencies on class is better solution.

File: packages/plugin-analytic-fb-pixel/README.md (hunk 2 above)
Comment (VojtaSim) [reply packages/plugin-analytic-fb-pixel/README.md]: 1. Following same pattern as in `ima-plugin-analytic-google`
2. `bind` is the only method that doesn't override default dependencies.

File: packages/plugin-analytic-fb-pixel/src/main.js (hunk 3 above)
Comment (VojtaSim) [reply packages/plugin-analytic-fb-pixel/src/main.js]: Ok. Will change it in google-analytic also as it's copy-pasted from it.

Comment (VojtaSim) [reply packages/plugin-analytic-fb-pixel/src/main.js]: - Added `$dependencies` to AbstractAnalytic, GoogleAnalytic and FacebookPixelAnalytic.
- removed `initBind` method from script and README. Class is registered when user requests it via `oc.get()` in his app.

**Pull request merged**